### PR TITLE
Move feature list from overview to abstract

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
Expands the abstract to include the highlights, but balanced against the overview.

File: spec/overview.md [hunk 1]
```diff
@@ -81,50 +61,102 @@ The following is a `tl;dr` summary of the specification summarizing how
       the previous entry.
     6. A [[ref: Data Integrity]] (DI) proof across the entry, signed by a DID
       Controller authorized to update the DIDDoc.
-4. In generating the first version of the DIDDoc, the DID Controller calculates
+2. In generating the first version of the DIDDoc, the DID Controller calculates
```
Comment (brianorwhatever) [inline spec/overview.md:64]:
```suggestion
1. In generating the first version of the DIDDoc, the DID Controller calculates
```

File: spec/overview.md [hunk 2]
```diff
@@ -81,50 +61,102 @@ The following is a `tl;dr` summary of the specification summarizing how
...
 An example of a `did:tdw` evolving through a series of versions can be seen in
 the [did:tdw Examples](#didtdw-example) of this specification.
 
 ### The `/whois` Use Case
 
-This DID Method introduces what we hope will be a widely embraced convention for all DID Methods -- the `/whois` path. This feature harkens back to the `WHOIS` protocol that was created in the 1970s to provide a directory about people and entities in the early days of ARPANET. In the 80's, `whois` evolved into a [[spec-inform:rfc920]] that has expanded into the [global whois](https://en.wikipedia.org/wiki/WHOIS) feature we know today as [[spec-inform:rfc3912]]. Submit a `whois` request about a domain name, and get back the information published about that domain.
+This DID Method introduces what we hope will be a widely embraced convention for
+all DID Methods -- the `/whois` path. This feature harkens back to the `WHOIS`
+protocol that was created in the 1970s to provide a directory about people and
+entities in the early days of ARPANET. In the 80's, `whois` evolved into a
```
Comment (brianorwhatever) [inline spec/overview.md:99]:
```suggestion
entities in the early days of ARPANET. In the 80's, `whois` evolved into
```

**Review by brianorwhatever — APPROVED** [review]

**Comment by swcurran** [comment]
Changes made.

**Review by brianorwhatever — APPROVED** [review]

**Pull request merged**